src/plugins/home/public/application/components/_solutions_section.scss

```diff
@@ -1,116 +1,99 @@
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.homSolutions {
   margin-top: -$euiSizeXL;
   min-height: $euiSize * 16;
-  display: flex;
 
-  .homSolutionsSection__panel {
-    display: flex;
-    align-items: stretch;
-
-    .homSolutionsSection__panelHeader {
-      border-radius: $euiBorderRadius 0 0 $euiBorderRadius;
-      color: $euiColorEmptyShade;
-
-      @include euiBreakpoint("xs", "s") {
-        border-radius: $euiBorderRadius $euiBorderRadius 0 0;
-      }
-    }
-
-    .homSolutionsSection__panelHeaderContent {
-      padding: $euiSize;
-      color: $euiColorGhost;
-    }
+  @include euiBreakpoint('xs', 's') {
+    flex-direction: column;
+  }
+}
 
-    .homSolutionsSection__panelIcon {
-      margin: 0 auto $euiSizeS;
-      padding: $euiSizeS;
-      background-color: $euiColorGhost;
-      box-shadow: none;
+.homSolutions__group {
+  max-width: 50%;
 
-      .euiIcon__fillNegative {
-        fill: $euiColorInk;
-      }
-    }
+  @include euiBreakpoint('xs', 's') {
+    max-width: none;
+  }
+}
 
-    .homSolutionsSection__panelSubtitle {
-      margin-top: $euiSizeXS;
-    }
+.homSolutionPanel {
+  align-items: stretch; // Necessary because element is a button
+  display: flex !important;
+  flex-direction: column;
+  overflow: hidden;
+}
 
-    .homSolutionsSection__panelDescriptions {
-      height: 100%;
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      padding: $euiSize;
+.homSolutionPanel__header {
+  color: $euiColorEmptyShade;
+  padding: $euiSize;
+}
 
-      @include euiBreakpoint("xs", "s") {
-        height: auto;
-        text-align: center;
-      }
-    }
-  }
+.homSolutionPanel__icon {
+  background-color: $euiColorEmptyShade !important;
+  box-shadow: none !important;
+  margin: 0 auto $euiSizeS;
+  padding: $euiSizeS;
+}
 
-  .homSolutionsSection__container {
-    flex-basis: $euiSizeXL * 4;
+.homSolutionPanel__subtitle {
+  margin-top: $euiSizeXS;
+}
 
-    &.homSolutionsSection__single {
-      max-width: 50%;
+.homSolutionPanel__content {
+  flex-direction: column;
+  justify-content: center;
+  padding: $euiSize;
 
-      @include euiBreakpoint("xs", "s") {
-        max-width: 100%;
-      }
-    }
+  @include euiBreakpoint('xs', 's') {
+    text-align: center;
   }
+}
 
-  .homSolutionsSection__multiple {
-    max-width: 50%;
+.homSolutionPanel__header {
+  background-color: $euiColorPrimary;
```

Comment:
I won't pepper this on each related change, but see the comment above regarding the solution card header design.

Reply:
Cool, left a comment above :)